### ISSUE TITLE
Excel Slicer – Slicer and controls in a slicer object cannot be read by NVDA

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15682]
### Steps to reproduce:

1. Turn on JAWS & NVDA screen reader.
2. Open Excel app in Win32.
3. Create a table (a few rows and columns of data, then press Ctrl+T to turn that into a table), select Design > Insert Slicer to insert slicers.)
4. Or open the attached test file (ExcelSlicersTestfile.xlsx).
5. Press Ctrl+Alt+5 to select the slicer object.
6. Press Tab or use arrow to navigate the controls within the slicer object.
7. Observe the issue.

**Test File**
[ExcelSlicersTestFile.xlsx](https://github.com/nvaccess/nvda/files/13164817/ExcelSlicersTestFile.xlsx)

### Actual behavior:
"Slicer and object controls" in a slicer object are not announced by NVDA. It remains silent while navigating through the slicer.

### Expected behavior:
"Slicer and object controls" in a slicer object should be announced by NVDA

### NVDA logs, crash dumps and other attachments:

https://github.com/nvaccess/nvda/assets/93735775/24c30965-cbac-4886-a334-32197ce94bfa

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by Adriani90** [comment]
This is also related to #6605, though in that issue we tried to access the slicer via f6 key, and there were also problems with the slicer manager and the slicer dialog in pivot tables.

**Comment by seanbudd** [comment]
Notably, this does work with UIA enabled.
